resources/metrics-template.md

```diff
@@ -29,6 +29,9 @@ Metric must be currently deployed/available, in contrast to a tool having the "p
 If there are several different ways to collect data for this metric, list them here. 
 This may include expressing a metric in different ways.
 
+### Ways the Metric Connects with Diversity, Equity, and/or Inclusion (Optional)
```

Comment:
We need to look at this holistically. -and/or- implies we can be selective about what we are connecting to.